layouts/projects/single.html

```diff
@@ -1,15 +1,29 @@
 {{ define "main" }}
     {{ partial "nav.html" (dict "nav_white" 1) }}
 
-    <div id="readingArea" class="row justify-content-center page-section mx-2" style="padding-top: 6.5rem;">
+    <div id="readingArea" class="px-4">
         <h1 class="fw-bold text-break" style="max-width: 100%;">{{ .Title }}</h1>
-        <h5 class="text-primary fst-italic">{{ .Params.subtitle }}</h5>
+        <h5 class="text-primary fst-italic mb-3">{{ .Params.subtitle }}</h5>
         <!-- Date layout string must follow the components listed here:
         https://gohugo.io/methods/time/format/#layout-string -->
-        <p class="font-monospace mb-3">
-            {{ .Date.Format "2 Jan 2006" }} | {{ .WordCount }} words | Written by {{ .Params.author }}
-        </p>
-        <hr>
+        <div class="project-metadata font-monospace fw-normal">
+            <span class="d-flex align-items-center gap-2">
+                <svg xmlns="http://www.w3.org/2000/svg" width="1.3em" height="1.3em" viewBox="0 0 24 24">
+                    <g fill="none">
+                        <path d="m12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.018-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z" />
+                        <path fill="#343a40" d="M21 12v7a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-7zm-5-9a1 1 0 0 1 1 1v1h2a2 2 0 0 1 2 2v3H3V7a2 2 0 0 1 2-2h2V4a1 1 0 0 1 2 0v1h6V4a1 1 0 0 1 1-1" />
+                    </g>
+                </svg>
+                <time>{{ .Date.Format "Jan 2, 2006" }}</time>
```

Comment:
```suggestion
                <time>{{ .Date.Format "2 Jan 2006" }}</time>
```
Inconsistant format. i think this is better, more malaysian way of expressing dates